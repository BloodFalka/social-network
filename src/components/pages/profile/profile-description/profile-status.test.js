import React from 'react'
import { create, act } from 'react-test-renderer'
import ProfileStatus from './profile-status.tsx'

describe('ProfileStatus', () => {
	test('status from props should be in the state', () => {
		const component = create(<ProfileStatus status="Kalen" />)
		const root = component.root
		act(() => root.props)
		expect(root.props.status).toBe('Kalen')
	})
})
