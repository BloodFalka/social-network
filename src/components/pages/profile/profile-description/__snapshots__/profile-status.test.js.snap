// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProfileStatus status from props should be in the state 1`] = `
<div
  className="status"
>
  <div
    className="status-text"
    onDoubleClick={[Function]}
  >
    Kalen
  </div>
</div>
`;
